app/merginapi.cpp

```diff
@@ -876,7 +885,11 @@ void MerginApi::updateInfoReplyFinished()
   r->deleteLater();
   mPendingRequests.remove( url );
   jsonDoc.setArray( fileArray );
-  downloadProjectFiles( projectFullName, jsonDoc.toJson( QJsonDocument::Compact ) );
+  QString info = QString( "STARTED - removed: %1, updated: %2, added: %3, renamed: %4" )
+                 .arg( files.value( "added" ).size() ).arg( files.value( "updated" ).size() )
+                 .arg( files.value( "removed" ).size() ).arg( files.value( "renamed" ).size() );
```

Comment:
Rather than creating info string and passing it to the next function to log it, why not just log it directly here?

And it would be good to also log the individual file names...?

Reply:
+1 for logging more info.
I also dont like passing extra parameter, but I think its better to log after a request is sent otherwise it would be logged but request wont be sent due to auth validation or api check. It might not happen often in this stage of data sync since it should validate and check api version while listing projects.